web-components/src/constants.ts

```diff
@@ -0,0 +1,15 @@
+// src/config/env.ts
+declare const __ENV__: {
+  isDevelopment: boolean
+}
+
+export const ENV = __ENV__
+
+export const ROBOTOFF_API_URL = "https://robotoff.openfoodfacts.org/api/v1"
+export const IS_DEVELOPMENT_MODE = ENV.isDevelopment
+export const PAGE_SIZE = 25
```

Comment:
The goal is to deploy those files as a static library. So we can't encode things like ROBOTOFF_API_URL or PAGE_SIZE this way.

Maybe we should use a specific web component for this kind of settings ? (or do you have a better idea), or a settings object ?